test/plugin/scenarios/springmvc-reactive-scenario/config/expectedData.yaml

```diff
@@ -44,7 +73,31 @@ segmentItems:
       startTime: not null
       endTime: not null
       componentId: 14
+      isError: false
+      tags:
+      - {key: url, value: not null}
+      - {key: http.method, value: GET}
```

Comment:
Does it miss the status_code  tag?

Reply:
@ascrutae  `status_code` is recorded When `httpStatus >= 400` in spring plugins.   `{GET}/testcase/error`  will `record status_code=500`
```java
            HttpStatus httpStatus = exchange.getResponse().getStatusCode();
            if (httpStatus != null && httpStatus.isError()) {
                span.errorOccurred();
                Tags.STATUS_CODE.set(span, Integer.toString(httpStatus.value()));
            }
```